### PR TITLE
comment out uploading to Play Store in upload_release build

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

File: Jenkinsfile.upload_release [hunk 1]
```diff
@@ -74,7 +74,7 @@ node ('macos1'){
     }
 
     stage('Deploy (Android)') {
-      sh ('bundle exec fastlane android release || exit 0')
+      //sh ('bundle exec fastlane android release || exit 0')
```
Comment (jeluard) [inline Jenkinsfile.upload_release:77]: Why not removing it?

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/WMog6E for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/WMog6E">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/dEBN8q for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/dEBN8q">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/F7ouaR for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/F7ouaR">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/SqRVeX for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/SqRVeX">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

File: Jenkinsfile.upload_release (hunk 1 above)
Comment (rasom) [reply Jenkinsfile.upload_release:77]: i believe we will need this sooner or later

Comment (jeluard) [reply Jenkinsfile.upload_release:77]: Makes sense. Could you add a comment detailing that? Something like `TODO: Reenable when stuff is available`

**Review by jeluard — APPROVED** [review]

File: Jenkinsfile.upload_release (hunk 1 above)
Comment (rasom) [reply Jenkinsfile.upload_release:77]: it is available actually

Comment (jeluard) [reply Jenkinsfile.upload_release:77]: I mean some comments detailing why it's commented but not removed :)

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/P4kQgU for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/P4kQgU">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/3KqdKF for branch PR-4252

<blockquote><div><strong><a href="https://i.diawi.com/3KqdKF">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Pull request closed without merging**